sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/CustomSingleClassificationTaskParameters.java

```diff
@@ -13,13 +13,13 @@
     /*
      * The projectName property.
      */
-    @JsonProperty(value = "projectName", required = true)
+    @JsonProperty(value = "project-name", required = true)
     private String projectName;
 
     /*
      * The deploymentName property.
      */
-    @JsonProperty(value = "deploymentName", required = true)
+    @JsonProperty(value = "deployment-name", required = true)
```

Comment:
this will get overwritten next time you regenerate, right?

Reply:
That is correct!